### PR TITLE
Docs setup

### PR DESCRIPTION
**Pull request by ArifPMan**

### CODE REVIEW

File: v6/postman/api_documentation/adding_and_verifying_custom_domains.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ To verify a domain, check the box to confirm you added the TXT and CNAME records
 
 If you choose not to verify the domain now, click the **Verify Later** button. 
 
-In addition if you want to delete the custom domain, click the "Delete Custom Domain" link.
+In addition if you want to delete the custom domain, click the "Changed2 Custom Domain" link.
```
Comment (numaanashraf) [inline v6/postman/api_documentation/adding_and_verifying_custom_domains.md]: Let's change this back to Delete itself

**Comment by numaanashraf** [comment]
@ArifPMan Looks like multiple docs issues are clubbed together in a single pull-request. This is not recommended as we cannot review individual issues separately and close them once they are ready.

File: v6/postman/launching_postman/installation_and_updates.md [hunk 2]
```diff
@@ -60,15 +81,15 @@ The native apps are not restricted by the Chrome standards for the menu bar. Wit
 
 ##### **Restricted headers**
 
-The latest versions of the native apps let you send headers like Origin and User-Agent. These are [restricted](/docs/v6/postman/sending_api_requests/interceptor_extension) in the Chrome app. 
+The latest versions of the native apps let you send headers like Origin and User-Agent. These are [restricted](https://www.getpostman.com/docs/v6/postman/sending_api_requests/interceptor_extension#restricted-headers) in the Chrome app.
```
Comment (numaanashraf) [inline v6/postman/launching_postman/installation_and_updates.md]: Let's not add the full absolute URL of the docs as this prevents us from rendering the same docs in multiple URLs (like the learning center and current docs) and makes it harder to test these with a local setup.

File: v6/postman/launching_postman/installation_and_updates.md [hunk 3]
```diff
@@ -60,15 +81,15 @@ The native apps are not restricted by the Chrome standards for the menu bar. Wit
...
+The latest versions of the native apps let you send headers like Origin and User-Agent. These are [restricted](https://www.getpostman.com/docs/v6/postman/sending_api_requests/interceptor_extension#restricted-headers) in the Chrome app. 
 
 ##### **Don't follow redirects option**
 
 This option exists in the native apps to prevent requests that return a 300-series response from being automatically redirected. Previously, users needed to use the Interceptor extension to do this in the Chrome app.
 
 ##### **Postman console**
 
-The latest version of the native app also has a built-in [console](/docs/v6/postman/sending_api_requests/debugging_and_logs), which allows you to view the network request details for API calls.
+The latest version of the native app also has a built-in [console](https://www.getpostman.com/docs/v6/postman/sending_api_requests/debugging_and_logs#network-calls-with-postman-console), which allows you to view the network request details for API calls.
```
Comment (numaanashraf) [inline v6/postman/launching_postman/installation_and_updates.md]: same comment as above. Let's not have absolute URLs.

File: v6/postman/launching_postman/installation_and_updates.md [hunk 4]
```diff
@@ -23,13 +23,34 @@ Once you’ve downloaded and unzipped the app, double click on Postman. You will
 
 ##### **Windows installation**
 
-   *   Download the setup file
+   *   Download the setup file
```
Comment (numaanashraf) [inline v6/postman/launching_postman/installation_and_updates.md]: There are new spaces at the end of the line. Is this intentional?

File: v6/postman/launching_postman/installation_and_updates.md [hunk 5]
```diff
@@ -23,13 +23,34 @@ Once you’ve downloaded and unzipped the app, double click on Postman. You will
...
+      Icon=/<Your_Install_Dir>/Postman/app/resources/app/assets/icon.png<br>
+      Terminal=false<br>
+      Type=Application<br>
+      Categories=Development;<br>
+
+   2. Check your desktop and double-click the Postman icon
+
+   **Note:** Ensure you specify the icon path as illustrated above. Postman introduced another folder 'app', one level up to the resources folder which did not exist earlier. 
+
+   If you are an Ubuntu 18 user, you must additionally install libconfig-2-4 package to ensure a smooth Postman run. This package, shipped by default until Ubuntu version 18 has been dropped, is mandatory for Postman run. Use the following command to install libconfig-2-4:
```
Comment (numaanashraf) [inline v6/postman/launching_postman/installation_and_updates.md]: This package, shipped by default until Ubuntu version 18, has been dropped and is mandatory for running Postman. Use the following command to install libconfig-2-4

File: v6/postman/launching_postman/installation_and_updates.md [hunk 6]
```diff
@@ -23,13 +23,34 @@ Once you’ve downloaded and unzipped the app, double click on Postman. You will
...
 
 ##### **Linux installation**
 
-   * Installation on Linux can vary between distributions. For installation on Ubuntu, download and unzip the file, then double click on Postman to open the app. 
+   * Installation on Linux can vary between distributions. For installation on Ubuntu, first download and unzip the file. And then create a desktop file by name postman.desktop. 
+
+   To create a desktop file, follow as illustrated below:
+
+   1. cat  > ~/.local/share/applications/postman.desktop
+      [Desktop Entry]<br>
```
Comment (numaanashraf) [inline v6/postman/launching_postman/installation_and_updates.md]: Let's use a fenced code block for this. https://github.github.com/gfm/#fenced-code-blocks

Adding `<br>` tags can help with formatting but prevents users from copy-pasting directly into a file.

File: v6/postman/launching_postman/installation_and_updates.md [hunk 7]
```diff
@@ -23,13 +23,34 @@ Once you’ve downloaded and unzipped the app, double click on Postman. You will
...
+      Type=Application<br>
+      Categories=Development;<br>
+
+   2. Check your desktop and double-click the Postman icon
+
+   **Note:** Ensure you specify the icon path as illustrated above. Postman introduced another folder 'app', one level up to the resources folder which did not exist earlier. 
+
+   If you are an Ubuntu 18 user, you must additionally install libconfig-2-4 package to ensure a smooth Postman run. This package, shipped by default until Ubuntu version 18 has been dropped, is mandatory for Postman run. Use the following command to install libconfig-2-4:
 
+   apt-get install libconfig-2-4
```
Comment (numaanashraf) [inline v6/postman/launching_postman/installation_and_updates.md]: Use a fenced code block for this too

```
apt-get install libconfig-2-4
```

File: v6/postman/scripts/postman_sandbox_api_reference.md [hunk 8]
```diff
@@ -88,7 +88,7 @@ The `pm.sendRequest` function allows sending HTTP/HTTPS requests asynchronously.
 
 Some things to know about `pm.sendRequest()`:
 
-* The method accepts a collection SDK compliant request and a callback. The callback receives 2 arguments, an error (if any) and SDK compliant response.
+* The method accepts a collection SDK compliant request and a callback. The callback receives 2 arguments, an error (if any) and SDK compliant response. Click [Response JSON](http://www.postmanlabs.com/postman-collection/Request.html#~definition) to view more information.
```
Comment (numaanashraf) [inline v6/postman/scripts/postman_sandbox_api_reference.md]: Refer [Collection SDK Documentation](link) for more information.

File: v6/postman/launching_postman/installation_and_updates.md (hunk 2 above)
Comment (ArifPMan) [reply v6/postman/launching_postman/installation_and_updates.md]: I changed the URL

File: v6/postman/launching_postman/installation_and_updates.md (hunk 4 above)
Comment (ArifPMan) [reply v6/postman/launching_postman/installation_and_updates.md]: I reviewed the spaces, I don't see it and when I viewed it in the following location it seems fine: http://127.0.0.1:4000/docs/v6/postman/launching_postman/installation_and_updates

File: v6/postman/launching_postman/installation_and_updates.md [hunk 9]
```diff
@@ -23,12 +22,68 @@ Once you’ve downloaded and unzipped the app, double click on Postman. You will
...
+
+#### Windows
+
+   Windows 7 and later are supported, older operating systems are not supported (and do not work).
+
+   Both ````ia32 (x86) and x64 (amd64)```` binaries are provided for Windows. Please note, the ARM version of Windows is not supported for now.
+
+### Linux
+   
+   The prebuilt binaries of Electron are built for Debian Jessie, but whether the prebuilt binary can run on a distribution depends on whether the distribution includes the libraries that Electron is linked to on the building platform, so only Debian Jessie is guaranteed to work, but following platforms are also verified to be able to run the prebuilt binaries of Electron:
```
Comment (numaanashraf) [inline v6/postman/launching_postman/installation_and_updates.md]: Let's change this to what Postman supports.

File: v6/postman/launching_postman/installation_and_updates.md [hunk 10]
```diff
@@ -23,12 +22,68 @@ Once you’ve downloaded and unzipped the app, double click on Postman. You will
...
+
+* Ubuntu 12.04 and later
+* Fedora 21
+* Debian 8 and later
+
+Electorn provides prebuilt binaries for following CPU architectures:
```
Comment (numaanashraf) [inline v6/postman/launching_postman/installation_and_updates.md]: Typo

**Pull request merged**